src/Form/UpdateForm.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Drupal\workspace\Form;
+
+use Drupal\Core\Ajax\AjaxResponse;
+use Drupal\Core\Ajax\CloseModalDialogCommand;
+use Drupal\Core\Ajax\PrependCommand;
+use Drupal\Core\Form\ConfirmFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Url;
+use Drupal\replication\Entity\ReplicationLogInterface;
+
+class UpdateForm extends ConfirmFormBase {
+
+  /**
+   * Get the current active workspace's pointer.
+   */
+  protected function getActive() {
+    $workspace = \Drupal::service('workspace.manager')->getActiveWorkspace();
+    $pointers = \Drupal::service('entity_type.manager')
+      ->getStorage('workspace_pointer')
+      ->loadByProperties(['workspace_pointer' => $workspace->id()]);
+    return reset($pointers);
+  }
+
+  /**
+   * Returns the upstream for the given workspace.
+   *
+   * @return \Drupal\multiversion\Entity\WorkspaceInterface
+   */
+  protected function getUpstream() {
+    $workspace = \Drupal::service('workspace.manager')->getActiveWorkspace();
+    if (isset($workspace->upstream)) {
+      return $workspace->upstream->entity;
+    }
+  }
+
+  /**
+   * {@inheritDoc}
```

Comment:
Let's keep this consistent with the Drupal core and use `{@inheritdoc}` instead (lowercase).